commodore/dependency_mgmt.py

```diff
@@ -95,7 +95,12 @@ def _read_components(
         if "version" in info:
             component_versions[component_name] = info["version"]
         else:
-            # Note: We use version=None as a marker for checking out the remote repo's default branch
+            cfg.register_deprecation_notice(
+                f"Component {component_name} doesn't have a version specified."
+                + "See https://syn.tools/commodore/reference/deprecation-notices.html#_components_without_versions for more details"
```

Comment:
I'm no Python expert, but isn't there a newline or whitespace missing between the 2 sentences or does Python do this automatically?

Reply:
It was missing, fixed in the latest version